modules/react/common/lib/utils/elements.ts

```diff
@@ -2,7 +2,7 @@
  * Is an element focusable? This function performs various tests to see if the element in question
  * can receive focus. Should skip disabled elements as they are not focusable.
  */
-export const isFocusable = (element: HTMLElement) => {
+export const isFocusable = (element: Element) => {
```

Comment:
Turns out I can make this type more permissive which helps for some callers that use `Element` instead of `HTMLElement`